Sources/PersonalMessage/Received.php

```diff
@@ -640,9 +642,9 @@ protected function getLabels(): void
 	 * @param int|string $limit Maximum number of results to retrieve.
 	 *    If this is left empty, all results will be retrieved.
 	 *
-	 * @return Generator<array> Iterating over the result gives database rows.
+	 * @return \Generator<array> Iterating over the result gives database rows.
```

Comment:
I think we could be more specific here with \Generator<int|string, Received>

Reply:
Static analysis does not like that because it changes the type then detected elsewhere.  Its correct to say its an array because elsewhere in that file we use it, we pass the $row into a `new self($row)`, which the constructor takes as the properties to load up the object.